docs/docs/native-docker.md

```diff
@@ -1,17 +1,64 @@
 ---
-title: Running Containers on Marathon
+title: Provisioning Containers
 ---
 
-Marathon supports Docker and Appc container images by either one of two runtimes; Docker Engine and Universal Container Runtime.
+# Provisioning Containers
 
-# Running Docker Containers on Marathon
+A containerizer is a Mesos agent component responsible for launching containers, within which you can run a Marathon app. Running apps in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically.
 
-Marathon enables users to run Docker container images with two different runtimes:
+Marathon enables users to run container images with two different runtimes:
 
-  1. [Docker containerizer](# Docker Containerizer) using the native Docker Engine as runtime.
-  2. Mesos containerizer using the Universal Container Runtime.
+1. Mesos containerizer using the [Universal Container Runtime](#ucr).
+1. [Docker containerizer](#docker-containerizer) using the native Docker Engine as runtime.
 
-# Docker Containerizer
+<a name="ucr"></a>
+
+# Universal Container Runtime
+
+The [Universal Container Runtime](http://mesos.apache.org/documentation/latest/container-image) (UCR) extends the Mesos containerizer to support provisioning [Docker](https://docker.com/) container images ([AppC](https://github.com/appc/spec) coming soon). This means that you can use both the Mesos containerizer and other container image types. You can still use the Docker container runtime directly ([instructions are below](#docker-containerizer)), but the Universal Container Runtime supports running Docker images without depending on the Docker Engine, which allows for better integration with Mesos.
+
+The following Marathon features _only_ work with the UCR:
+
+- [Pods]({{ site.baseurl }}/docs/pods.html).
+- GPUs.
+- [Authentication to a private Docker registry using a secret store]({{ site.baseurl }}/docs/native-docker-private-registry,html).
+
+## Provisioning Containers with the UCR
+
+To provision containers with the UCR, specify the container type `MESOS` and a the appropriate object in your application definition. Here, we specify a Docker container with the `docker` object.
+
+The Mesos containerizer provides a `credential`, with a `principal` and an optional `secret` field to authenticate when downloading the Docker image.
+
+```json
+{  
+   "id":"mesos-docker",
+   "container":{  
+      "docker":{  
+         "image":"mesosphere/inky",
+         "credential":{  
```

Comment:
This does not seem correct. The credential field is deprecated. We use a new secret field called `config` to provide image pulling credentials. Please check with @ichernetsky or @jdef on the Marathon API.